go/client/logclient_test.go

```diff
@@ -229,8 +229,8 @@ func TestAddChainRetries(t *testing.T) {
 	const leeway = time.Millisecond * 100
 	const leewayRatio = 0.1 // 10%
 	const instant = time.Millisecond
-	const fiveSeconds = time.Second * 5
-	const sevenSeconds = time.Second * 7 // = 1 + 2 + 4
+	const five = time.Second * 5
+	const seven = time.Second * 7 // = 1 + 2 + 4
```

Comment:
Removed the constants and converted `deadlineLength`; `retryAfter` isn't ever used as a `time.Duration` though.